_episodes/036-luca-palmieri.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "From Zero to Production"
```

Comment:
Let's mention Luke's name in the title too.

Reply:
I'd still like this to be update.

Reply:
Any suggestions?

Reply:
> From Zero to Production with Luca Palmieri